packages/webview_flutter/example/test_driver/webview.dart

```diff
@@ -456,6 +457,69 @@ void main() {
       isPaused = await controller.evaluateJavascript('isPaused();');
       expect(isPaused, _webviewBool(false));
     });
+
+    test('Video plays inline when allowsInlineMediaPlayback is true', () async {
+      if (Platform.isIOS) {
```

Comment:
This test should run on all platforms (so that we can be sure future plugin / platform versions don't break).

Can you clearly observe a different behavior on Android? If so, perhaps this could be a separate else branch to verify that.